From b7365143f43317aa537d6238ddb5c0d0b707e844 Mon Sep 17 00:00:00 2001
From: jerome <jerome.anand@intel.com>
Date: Wed, 24 Apr 2013 12:41:47 +0530
Subject: [PORT FROM MAIN] [REMOTE BGM][DO NOT UPSTREAM] Local audio routes to remote sink when playing video only stream on remote sink

BZ: 101834

The audio policy routing to the remote sink gets initialized
only on first successful playback of an audio stream. In case
of video only stream, the audio path is not hit. So this
information is transmitted to HAL and retrieved in APM for
routing the audio to appropriate sinks.

Change-Id: I6f806cbd2c4a2271914bbebd30cd005fbda317f5
Orig-Change-Id: I5a776a6913b22db251c17b5d840c419b2c37d42c
Signed-off-by: jerome <jerome.anand@intel.com>
Reviewed-on: http://android.intel.com:8080/104287
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>
Reviewed-by: Sikkandar D, Madar <madar.sikkandar.d@intel.com>
Reviewed-by: Krishna, Samaga <samaga.krishna@intel.com>
Tested-by: Joshi, SantoshX <santoshx.joshi@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 include/hardware/audio.h     |    1 +
 modules/audio/audio_policy.c |    1 -
 2 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index d51c571..3fa3521 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -112,6 +112,7 @@ __BEGIN_DECLS
 
 /* Remote BGM state */
 #define AUDIO_PARAMETER_KEY_REMOTE_BGM_STATE "bgm_state"
+#define AUDIO_PARAMETER_VALUE_REMOTE_BGM_AUDIO "bgm_audio"
 
 /* Context awareness */
 #define AUDIO_PARAMETER_KEY_CONTEXT_AWARENESS_STATUS "context_awareness_status"
diff --git a/modules/audio/audio_policy.c b/modules/audio/audio_policy.c
index 0496b09..309cb56 100644
--- a/modules/audio/audio_policy.c
+++ b/modules/audio/audio_policy.c
@@ -238,7 +238,6 @@ static bool ap_is_offload_supported(struct audio_policy *pol,
                                    bool isVideo,
                                    bool isStreaming)
 {
-    ALOGV("legacy in use. ap_is_offload_supported");
     return 0;
 }
 
-- 
1.7.4.1

